client/src/components/Search/Search.js

```diff
@@ -444,40 +444,42 @@ class Search extends Component {
     const clearable = (Object.keys(data).length > 0);
 
     return (
-      <Focusedzone onClickOut={this.show} className="search">
-        <SearchBox
-          {...props}
-          name={`SearchBox__${name}`}
-          onChange={this.handleChange}
-          onSearch={this.doSearch}
-          onToggleFilter={this.toggle}
-          onHideFilter={this.show}
-          onHide={this.hide}
-          onClear={this.clearSearchBox}
-          searchText={searchText}
-          hideable={hideable}
-          expanded={expanded}
-          id={`${id}_searchbox`}
-          showFilters={Boolean(forceFilters || formSchemaUrl)}
-          dirty={dirty}
-          clearable={clearable}
-          onTagDelete={this.clearFormFilter}
-          onTagClick={this.focusFormFilter}
-          tagData={this.formatTagData()}
-        >
-
-          <SearchForm
-            id={formId}
-            identifier={identifier}
-            visible={visible}
-            expanded={expanded}
-            formSchemaUrl={formSchemaUrl}
+      <div ref={node => { this.nodeRef = node; }}>
```

Comment:
Don't add a new element to the component - this might mess with css customisations.
Instead, use [forwarding refs](https://legacy.reactjs.org/docs/forwarding-refs.html) and forward a ref from this component down to the `FocusedZone` component.

Reply:
Updated.